### PR TITLE
feat: add member_map skel

### LINKED ISSUE #25: [honeybadger] define member structure
**Subject**

**Detail**

**Reference**

### PR DESCRIPTION
**Pull request by zeroFruit**
## Related Issue

resolve: #25 

## Description

Add `member_map` skeleton

## Checklist

Thank you for your contribution.
Before submitting this PR, please make sure:

- [ ] Test case
- [x] End of Work

### CODE REVIEW

**Comment by AgwaB** [comment]
plz apply goimports

File: member_map_test.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package cleisthenes_test
+
+import "testing"
+
+func TestMember_Address(t *testing.T) {
+	// implement me :-(
+}
+
+func TestMemberMap_CRD(t *testing.T) {
```
Comment (AgwaB) [inline member_map_test.go:9]: test all at one test?

File: member_map.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	lock    sync.RWMutex
+	members map[MemberId]*Member
+}
+
+func NewMemberMap() *MemberMap {
+	panic("implement me w/ test case :-)")
+}
+
+// GetAllMembers returns current members into array format
+func (m *MemberMap) GetAll() []Member {
```
Comment (hihiboss) [inline member_map.go]: In **Go** convention, `Get` is not used. How about `func AllMembers()`?

File: member_map.go [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	panic("implement me w/ test case :-)")
+}
+
+func (m *MemberMap) Get(id MemberId) Member {
```
Comment (hihiboss) [inline member_map.go]: In Go convention, Get is not used. How about `func Member(id MemberId)`?

**Review by hihiboss — COMMENTED** [review]
My comments are just about Go convention :)

**Review by boohyunsik — DISMISSED** [review]
LGTM :-)

File: member_map.go (hunk 2 above)
Comment (AgwaB) [reply member_map.go]: I agree

File: member_map.go (hunk 3 above)
Comment (AgwaB) [reply member_map.go]: I agree

**Review by AgwaB — APPROVED** [review]

**Review by hihiboss — APPROVED** [review]
LGTM :)

**Pull request merged**